app/src/js/obj-validation.js

```diff
@@ -86,13 +86,16 @@ Bolt.validation = (function () {
      */
     function showAlertbox(id, field, msg) {
         var alertbox;
+        var fieldname = $('label[for="' + field.id + '"]').contents().first().text().trim() || field.name;
+
+        fieldname = fieldname.replace(/(.*):+$/g, "$1" );
```

Comment:
I might be missing something here … but why are we even getting these colons in the first place? Can we attack root cause?